fanfic_reader/client/websocket_reader.js

```diff
@@ -189,6 +190,29 @@ function setRemoteUsername(username) {
     }
 }
 
+/**
+ * Writes marked paragraph indices to localStorage in JSON format
+ * @param {Array} array
+ */
+function storeMarkedP(array) {
+    const myJSON = JSON.stringify(array);
+    window.localStorage.setItem('markedParagraphs', myJSON);
+    console.log('localStorage markedParagraphs:', localStorage.getItem('markedParagraphs'))
+}
+
+/**
+ * Checks if the user has marked paragraphs in localStorage, and loads them
+ * @returns 
+ */
+function checkForLocalMarkedParagraphs() {
+    if (window.localStorage.getItem('markedParagraphs')) {
+        markedDialogue = JSON.parse(window.localStorage.getItem('markedParagraphs'));
+        console.log('Local marked paragraph data found. markedDialogue array set. markedDialogue now:', markedDialogue);

```

Comment:
Generally we don't want to leave console logs hanging around unless they have some practical value for the published version